homeassistant/components/statistics/sensor.py

```diff
@@ -131,6 +132,31 @@
     STAT_DATETIME_OLDEST,
 )
 
+# Statistics which retain the unit of the source entity
+STAT_NUMERIC_RETAIN_UNIT = (
```

Comment:
Use {} so it's a set instead of a tuple so it doesn't have to be searched linearly

Reply:
Now I understand your "set()" comment from before!!! Yes that makes a lot of sense, of course.

Reply:
Done